app/models/data-operation.ts

```diff
@@ -298,17 +345,30 @@ export default class DataOperation {
    * @param collection - The CMR ID of the collection being operated on
    * @param vars - An array of objects containing variable id and name
    * @param granules - An array of objects containing granule id, name, and url
+   * @param varUrlTypes - the types of variable related URLs to include in the source
    */
   addSource(
     collection: string,
     vars?: CmrUmmVariable[],
     granules?: HarmonyGranule[],
+    varUrlTypes: CmrRelatedUrlType[] = [CmrRelatedUrlType.ColorMap],
   ): void {
-    const variables = vars ? vars.map(({ umm, meta }) => ({
-      id: meta['concept-id'],
-      name: umm.Name,
-      fullPath: umm.Name,
-    })) : undefined;
+    const variables = vars ? vars.map(({ umm, meta }) => {
+      const schemaVar: HarmonyVariable = {
+        id: meta['concept-id'],
+        name: umm.Name,
+        fullPath: umm.Name,
+      };
+      if (umm.RelatedURLs) {
+        const filteredUrls = umm.RelatedURLs
+          .filter((url) => varUrlTypes.includes(url.Type as CmrRelatedUrlType))
```

Comment:
So we're ignoring any urls that do not include a `Type` attribute? What's the rationale?

Reply:
It's actually not filtering based on the existence of the Type property. It's just ignoring any urls that don't have Type = any of the types specified in the `varUrlTypes` argument. The thinking was to make the default argument be `[CmrRelatedUrlType.ColorMap]` since that's the only behavior that we want for now.

Reply:
I could go either way on this, but I could see this tripping us up in the future if we assume "Oh yeah, we're passing on related URLs, so you can put that info there"

Also if we want to pass on the minimum, we technically only want to pass on ColorMap URLs if they have a subtype of "GIBS"

Reply:
> Also if we want to pass on the minimum, we technically only want to pass on ColorMap URLs if they have a subtype of "GIBS"

Good to know. 

My thinking was that in case someone finds a use case that results in the relatedUrls field being abused, we'd not want the data operation to blow up without our knowledge.

Reply:
Passing all related URLs now.